packages/back-end/src/app.ts

```diff
@@ -254,6 +257,38 @@ app.options(
   }
 );
 
+//API routes that require an access token
+app.get(
+  "/api/v1/healthcheck",
+  validateAccessTokenApiReq(),
+  apiController.getHealthCheck
+);
+app.get(
+  "/api/v1/features/:featureId",
+  [validateAccessTokenApiReq()],
+  apiController.getFeatureApi
+);
+app.get(
+  "/api/v1/features",
+  [validateAccessTokenApiReq()],
+  apiController.listFeaturesApi
+);
+app.post(
+  "/api/v1/features/:featureId",
+  [validateAccessTokenApiReq()],
+  apiController.postFeatureApi
+);
+app.put(
+  "/api/v1/features/:featureId",
+  [validateAccessTokenApiReq()],
+  apiController.putFeatureApi
+);
+app.delete(
+  "/api/v1/features/:featureId",
+  [validateAccessTokenApiReq()],
+  apiController.deleteFeatureApi
+);
```

Comment:
I would recommend using Express [Routers](https://expressjs.com/en/5x/api.html#router) to reduce redundancy and the chance of human error with regards to the requirement that `validateAccessTokenApiReq` always be run.

For example, you can create a router as follows (e.g. `./routers/api/v1Router.ts`):

```ts
import express from 'express'

const router = express.Router()

router.use(validateAccessTokenApiMiddleware)

router.get("/healthcheck", apiController.getHealthCheck)
router.get("/features/:featureId", apiController.getFeatureApi)

export { router as v1Router } 
```

And in the `app.ts` file:

```js
import { v1Router } from './routers/api/v1Router'

app.get("/api/v1", v1Router)
```

I find the usage of routers allows developers to reduce redundancy and avoid human error. The `app.ts` file is getting quite large, I would recommend breaking it out into routers.

It may also make sense for the public API to be its own router, with nested routers for each version as well.